courses/free-courses-ko.md

```diff
@@ -153,12 +155,12 @@
 * [AngularJS](https://www.youtube.com/playlist?list=PLs_XsVQJKaBk_JN5RctLmmVrGwEzpzqaj) - 양재동 코드랩 (YouTube)
 
 
-#### Node.js
+#### <a id="node.js"></a>Node.js
```

Comment:
change the index link to `#nodejs` instead

Reply:
Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc